include/configs/imx8mm_evk.h

```diff
@@ -10,7 +10,11 @@
 #include <asm/arch/imx-regs.h>
 #include "imx_env.h"
 
+#ifdef CONFIG_SPL_WITH_DDR_FIRMWARE
+#define CONFIG_SPL_MAX_SIZE		(244 * 1024)
```

Comment:
Could you leave a comment how you calculated that value (244k)

Reply:
Great point.  I will add.  The difference here was re-adding the size of the DDR firmware (~96KB).  Normally, the u-boot build would limit the SPL size to assume the DDR firmware was going to be added in the later step.  But it still needs to fit in TCM.

Reply:
Is there a soc-family related header file that could define this value? Here you're only adding to evk.h, but the same will be required for other boards based on the same soc family.

Reply:
Great point.  This is defined in each of the include/config for:
```
$ fgrep -nr "CONFIG_SPL_MAX_SIZE" . | grep mx8m
./include/configs/imx8mp_evk.h:15:#define CONFIG_SPL_MAX_SIZE		(152 * 1024)
./include/configs/imx8mq_evk.h:13:#define CONFIG_SPL_MAX_SIZE		(148 * 1024)
./include/configs/imx8mm_ab2.h:13:#define CONFIG_SPL_MAX_SIZE		(148 * 1024)
./include/configs/verdin-imx8mp.h:14:#define CONFIG_SPL_MAX_SIZE				(152 * 1024)
./include/configs/imx8mq_val.h:14:#define CONFIG_SPL_MAX_SIZE		(148 * 1024)
./include/configs/imx8mm_val.h:14:#define CONFIG_SPL_MAX_SIZE		(148 * 1024)
./include/configs/verdin-imx8mm.h:16:#define CONFIG_SPL_MAX_SIZE		(148 * 1024)
./include/configs/imx8mn_evk.h:14:#define CONFIG_SPL_MAX_SIZE		(208 * 1024)
./include/configs/imx8mn_ab2.h:14:#define CONFIG_SPL_MAX_SIZE		(208 * 1024)
./include/configs/imx8mm_evk.h:14:#define CONFIG_SPL_MAX_SIZE		(148 * 1024)
```

Reply:
In a perfect world this would be an actual Kconfig and like you said it could be set by SoC.  What a mess to implement tho.